projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2022, My Name
```

Comment:
Shouldn't the license file be modified to fit the overall vdk project, given that the extension essentially becomes public the moment it is merged?
cc: @tozka

Reply:
Yes, please remove the license file as well. As I mentioned before. Remove unnecessary files before merging.